lightning-background-processor/src/lib.rs

```diff
@@ -764,34 +758,44 @@ mod tests {
 	}
 
 	#[test]
-	fn test_not_handling_events_until_until_graph_sync_completion() {
-		let mut nodes = create_nodes(2, "test_not_handling_events_until_until_graph_sync_completion".to_string());
+	fn test_not_pruning_network_graph_until_graph_sync_completion() {
+		let mut nodes = create_nodes(2, "test_not_pruning_network_graph_until_graph_sync_completion".to_string());
 		let channel_value = 100000;
 		let data_dir = nodes[0].persister.get_data_dir();
 		let persister = Persister::new(data_dir.clone());
+		let network_graph = nodes[0].network_graph.clone();
+		let features = ChannelFeatures::empty();
+		network_graph.update_channel_from_partial_announcement(42, 53, features, nodes[0].node.get_our_node_id(), nodes[1].node.get_our_node_id());
+		let original_graph_description = network_graph.to_string();
+		assert!(original_graph_description.contains("42: features: 0000, node_one:"));
 
-		// Set up a background event handler for FundingGenerationReady events.
 		let (sender, receiver) = std::sync::mpsc::sync_channel(1);
 		let event_handler = move |event: &Event| {
 			sender.send(handle_funding_generation_ready!(event, channel_value)).unwrap();
```

Comment:
You can use an empty event handler since you don't need to test that logic here.

Reply:
Oops, good point.